src/PAModel/CanvasDocument.cs

```diff
@@ -431,20 +431,58 @@ private FilePath GetAssetFilePathWithoutPrefix(string path)
             return FilePath.FromMsAppPath(path.Substring(AssetFilePathPrefix.Length));
         }
 
-        private void StabilizeAssetFilePaths()
+        internal void StabilizeAssetFilePaths(ErrorContainer errors)
         {
             _entropy.LocalResourceFileNames.Clear();
 
+
+            // If a name matches caseinsensitive but not casesensitive, it is a candidate for rename
+            var caseInsensitiveNames = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+            var caseSensitiveNames = new HashSet<string>(StringComparer.Ordinal);
+            foreach (var resource in _resourcesJson.Resources.OrderBy(resource => resource.Name, StringComparer.Ordinal))
+            {
+                if (resource.ResourceKind != ResourceKind.LocalFile)
+                    continue;
+
+                if (!caseInsensitiveNames.Contains(resource.Name))
+                {
+                    caseInsensitiveNames.Add(resource.Name);
```

Comment:
nit: Add() returns a boolean, so I think this could be: 

if (caseInsensitiveNames.Add(resource.Name))
{
 caseSensitiveNames.Add(resource.Name);
}